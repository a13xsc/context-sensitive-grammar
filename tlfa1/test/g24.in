   (                  {  l       ,       U      ,       S     ,z          ,   >,       O        ,         X         ,        `     ,a        ,        F         ,      :,    "     ,      A     ,C        ,  u          ,        m  ,      P          ,       W        ,         Y , ~ ,  -     ,        H  ,          <         ,      B    ,       !      ,      $     ,    L    ,I       ,       N        ,       i ,     _,   V        ,      Z    ,      6  ,       D    ,        R,T          ,         M      , q    ,   E      , K   ,       Q        ,       s     ,       G         } ,         {          _        ,         ~   ,      l          ,       `        ,          q   , a  , " ,      >    ,      m   ,      u, :    ,     !   ,        -       , i   ,z      ,        $       ,<    ,     6  ,   s              }      ,          { (   A    ,    _zICsTqKYl!NK_NLmT      )    ,     (          A          ,    ><~mi )       ,(        F    ,        TR-RMZMSCU:L~        )       ,   (   H        ,  !ZU"BH><T~Z`GMQl>l     )        ,     (   H ,         e        )     , (     K      ,     ">`musq-l" )         ,      (   K,  X:SN_<quTV    ),   (       M     ,WILD66HNY         )          ,     (      L          ,       aFsKBW       )    ,      (   L   ,   NLu-NWCuBOQPB      )        ,       (         L   ,e         )      ,  ( L          ,   S_`LE )        ,        (    N      ,     CSUGszKzIE  )    ,(         N ,    >:l )       ,   (      S   ,   aKq:zBAZMYm_Si-          ) ,  ( S,u"umamuqlsss6~mqum )  ,        (       U   ,        ~ELNP        )        ,          ( T    ,     SOBq:   ),      (        T   ,a>$m>_   )     ,      (        W    , LLLLZZZLZZLLL )        ,        (   Y      , SqLlPs`RqGHOHMFA  )   ,   (        Y     ,         _CX~DLBXWDq:V-CQ:z   )          ,          (     X  ,   WWLZWLLL )      ,   (         Z        ,    GW        )    ,   (      Z  ,   LLLLLLLLLLLL     )       }              ,        U     )
