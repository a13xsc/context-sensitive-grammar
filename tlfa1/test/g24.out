M
D
G
I
F
C
O
U
B
Y
Q
E
V
R
P
